src/cli/configure/upgrade/patches/7.3.1/index.ts

```diff
@@ -0,0 +1,14 @@
+import { tryPatchRenovateConfig } from '../../../patchRenovateConfig';
+
+import { tryAddEmptyExports } from './addEmptyExports';
+import { tryPatchDockerfile } from './patchDockerfile';
+import { tryPatchServerListener } from './patchServerListener';
+
+export const upgrade = async () => {
+  await Promise.all([
+    tryAddEmptyExports(),
+    tryPatchRenovateConfig(),
+    tryPatchDockerfile(),
+    tryPatchServerListener(),
```

Comment:
This is a bit scope creepy, but thoughts on establishing a convention that patches need to describe what they are doing, e.g. which files are affected, how, and why? This could involve either or both of:

- Console logging
- Commit messaging that is propagated to the autofix

Happy to raise this (creating guidelines for contributing patches) as a separate issue if we agree it's a good idea, doesn't have to be solved in this initial PR 👍